tls/linkerd-client.yml

```diff
@@ -15,5 +15,5 @@ routers:
     # initiate TLS on all outgoing connections it creates.
     tls:
       trustCerts:
-      - # FIXME
-      commonName: # FIXME
+      - /ca-chain.cert.pem
```

Comment:
I kind of like the `#FIXME`s here to indicate where the user should edit the configs. And if the filled out sections are in the READMEs they don't also need to be filled in here?

Reply:
ah! you're right, i did not mean to make these edits